searchsploit: edit page (#12310)
tldr-pages/tldr: pages/common/searchsploit.md

# searchsploit

> > Search Exploit Database's database for exploits, shellcodes and/or papers.

Reviewer: What do you guys think?

Reviewer 2: ```suggestion
> Search Exploit's database for exploits, shellcodes and/or papers.
```
Maybe this?

`searchsploit sudo 1.8.27`

> - Show the exploit-db link (--[w]ww) to the found resources:

Reviewer: I didn't find a better way to mention the short option

Reviewer 2: ```suggestion
- Show the exploit-db link to the found resources:
```
I don't think the short option is required here. If you really need it, then you can go with this:

```suggestion
- Show the exploit-db link to the found resources ([w]):
```

`searchsploit --www {{search_terms}}`

> - Copy ([m]irror) the resource to the current directory (requires the number of the exploit):

Reviewer: I think it's good to still leave copy as it's more understandable

`searchsploit --www {{search_terms}}`

- Copy ([m]irror) the resource to the current directory (requires the number of the exploit):

`searchsploit --mirror {{exploit_number}}`

- E[x]amine the resource, using the pager defined in the `$PAGER` environment variable:

`searchsploit --examine {{exploit_number}}`

- [u]pdate the local exploit database:

`searchsploit --update`

> - Search for a [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:

Reviewer 2: ```suggestion
- Search for the [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:
```

> - Open the resource to read with the pager defined in the `$PAGER` environment variable:

Reviewer: We can highlight the short option "-x":

```suggestion
- Open (e[x]amine) the resource, reading with the pager defined in the `$PAGER` environment variable:
```

Or even:

```suggestion
- E[x]amine the resource, using the pager defined in the `$PAGER` environment variable:
```

`searchsploit --examine {{exploit_number}}`

> - Update the local exploit database:

Reviewer: ```suggestion
- [u]pdate the local exploit database:
```
We can add this missing mnemonic

`searchsploit --update`

> - Search for [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:

Reviewer: ```suggestion
- Search for a [c]ommon [v]ulnerabilities and [e]xposures (CVE) value:
```
